notebooks/working/reproducing_deconfounder_results.ipynb

```diff
@@ -0,0 +1,1680 @@
+       "</tr>\n",
+       "<tr>\n",
+       "  <th>Kurtosis:</th>       <td>15.259</td>  <th>  Cond. No.          </th>  <td>    1.04</td> \n",
+       "</tr>\n",
+       "</table><br/><br/>Warnings:<br/>[1] Standard Errors assume that the covariance matrix of the errors is correctly specified."
+      ],
+      "text/plain": [
+       "<class 'statsmodels.iolib.summary.Summary'>\n",
+       "\"\"\"\n",
+       "                                 OLS Regression Results                                \n",
+       "=======================================================================================\n",
+       "Dep. Variable:                      y   R-squared (uncentered):                   0.002\n",
+       "Model:                            OLS   Adj. R-squared (uncentered):              0.002\n",
+       "Method:                 Least Squares   F-statistic:                              28.11\n",
+       "Date:                Wed, 29 Apr 2020   Prob (F-statistic):                    6.41e-13\n",
+       "Time:                        19:50:40   Log-Likelihood:                         -50073.\n",
+       "No. Observations:               23961   AIC:                                  1.001e+05\n",
+       "Df Residuals:                   23959   BIC:                                  1.002e+05\n",
+       "Df Model:                           2                                                  \n",
+       "Covariance Type:            nonrobust                                                  \n",
+       "==============================================================================\n",
+       "                 coef    std err          t      P>|t|      [0.025      0.975]\n",
+       "------------------------------------------------------------------------------\n",
+       "TOTALEXP      -0.0944      0.013     -7.464      0.000      -0.119      -0.070\n",
+       "marital       -0.0130      0.013     -1.031      0.303      -0.038       0.012\n",
+       "==============================================================================\n",
+       "Omnibus:                     4949.500   Durbin-Watson:                   2.004\n",
+       "Prob(Omnibus):                  0.000   Jarque-Bera (JB):           150337.860\n",
+       "Skew:                          -0.278   Prob(JB):                         0.00\n",
+       "Kurtosis:                      15.259   Cond. No.                         1.04\n",
+       "==============================================================================\n",
+       "\n",
+       "Warnings:\n",
+       "[1] Standard Errors assume that the covariance matrix of the errors is correctly specified.\n",
+       "\"\"\""
+      ]
+     },
+     "execution_count": 39,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "mod_no = smf.ols('y ~ -1 + TOTALEXP + marital ', data = df)\n",
+    "results_no = mod_no.fit()\n",
+    "results_no.summary()"
```

Comment:
Am I understanding this correctly, that the model without the confounder_PPCA had lower bias in coefficients than the model with the inferred confounder_PPCA?

Can you make this expicit (by combining the resulting coefficients in a single dataframe showing "true", "no_confounder_ppca", "with_confounder_ppca"? And perhaps adding a statement in text about this?